### ISSUE TITLE
External Editor isn't visible

### ISSUE THREAD

**Issue by t1gu1** [flathub/com.axosoft.GitKraken#156]

**Comment by Lctrs** [comment]
> I guess this is a problem with the flatpak univers?

Exactly. Flatpak runs in a sandbox so we don't see other installed flatpaks or programs installed on the host.

**Comment by bencat-sixense** [comment]
I'm facing the same issue.
There is no way to customize the command that GK use to launch it ?

**Comment by sebgott** [comment]
I also have the same issue. Would be nice to see this addressed somehow.

**Comment by Lctrs** [comment]
There is not much we can do at our level.

**Comment by t1gu1** [comment]
@bencat-sixense (Or anyone)

I didn't have an environnement with linux available at the moment.
I would be curious if something like Flatseal could give permissions to make that works?

https://github.com/tchx84/flatseal?tab=readme-ov-file
